db/migrations/versions/b414b1fbe915_.py

```diff
@@ -0,0 +1,93 @@
+"""empty message
+
+Revision ID: b414b1fbe915
+Revises:
+Create Date: 2022-08-08 14:23:57.145073
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "b414b1fbe915"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "applications",
+        sa.Column(
+            "id",
+            sqlalchemy_utils.types.uuid.UUIDType(binary=False),
+            nullable=False,
+        ),
+        sa.Column("account_id", sa.String(), nullable=False),
+        sa.Column("round_id", sa.String(), nullable=False),
+        sa.Column("fund_id", sa.String(), nullable=False),
+        sa.Column("project_name", sa.String(), nullable=True),
+        sa.Column(
+            "started_at",
+            sa.DateTime(),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column(
+            "status",
+            postgresql.ENUM(
```

Comment:
Don't think this type will work with Sqlite? So I think we'd either have to fully drop sqlite support, or change it to a simple varchar.